src/utils/navigation.ts

```diff
@@ -57,7 +57,12 @@ export const NAVIGATION: Navigation = {
   [config.UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_URL]: {
     currentPage: config.UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_PAGE,
     previousPage: () => config.OVERSEAS_ENTITY_PRESENTER_URL,
-    nextPage: [config.WHO_IS_MAKING_UPDATE_PAGE, config.UPDATE_NO_CHANGE_BENEFICIAL_OWNER_STATEMENTS_PAGE]
+    nextPage: [config.WHO_IS_MAKING_UPDATE_PAGE, config.OVERSEAS_ENTITY_PAYMENT_WITH_TRANSACTION_URL]
```

Comment:
Is config.OVERSEAS_ENTITY_PAYMENT_WITH_TRANSACTION_URL next?  It's a sort of next for UPDATE_REVIEW_STATEMENT_URL but not for UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_URL.